### PR TITLE
fix: toolType to toolName

### PR DESCRIPTION
**Pull request by jlopes90**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Fix

* **What is the current behavior?** (You can also link to an open issue here)
Measurement Added and Removed, toolType change to toolName.
Because Modified and Completed is toolName.

* **What is the new behavior (if this is a feature change)?**
No

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Other information**:
No

### CODE REVIEW

File: src/stateManagement/toolState.js [hunk 1]
```diff
@@ -30,18 +30,18 @@ function getElementToolStateManager(element) {
...
+function addToolState(element, toolName, measurementData) {
   const toolStateManager = getElementToolStateManager(element);
 
-  toolStateManager.add(element, toolType, measurementData);
+  toolStateManager.add(element, toolName, measurementData);
 
   const eventType = EVENTS.MEASUREMENT_ADDED;
   const eventData = {
-    toolType,
+    toolName,
```
Comment (JamesAPetts) [inline src/stateManagement/toolState.js]: This is a breaking api change. I'm happy to change this as its technically correct, although perhaps we should have a deprecated alias with `toolType` Either that or we have its v4 only. What do you think @dannyrb ?

Comment (dannyrb) [reply src/stateManagement/toolState.js]: I think v4 is coming down the pipe soon enough that we should just do it there, and make a note of it in our breaking changes / migration guide.

**Comment by dannyrb** [comment]
@jlopes90 can you make this same change on our `vNext` branch?

**Comment by jlopes90** [comment]
> @jlopes90 can you make this same change on our `vNext` branch?

At the moment, I will not have the time to make the change, but.

**Comment by jlopes90** [comment]
> @jlopes90 can you make this same change on our `vNext` branch?

At this moment! I'm busy, I have things to do with my project. So I'll see if I can.

**Comment by jlopes90** [comment]
Sorry, I was wrong to click close pull request

**Comment by jlopes90** [comment]
> @jlopes90 can you make this same change on our `vNext` branch?

How do I do it?

**Comment by jlopes90** [comment]
Can I change in vNext branch?

**Comment by mikehazell** [comment]
@dannyrb This can be closed now. #1216 fixed this issue.

**Pull request closed without merging**